code/game/area/Space Station 13 areas.dm

```diff
@@ -77,7 +77,7 @@ proc/process_ghost_teleport_locs()
 			ghostteleportlocs += AR.name
 			ghostteleportlocs[AR.name] = AR
 		var/turf/picked = safepick(get_area_turfs(AR.type))
-		if (picked && (picked.z == 1 || picked.z == 5 || picked.z == 3))
+		if (picked && (picked.z == ZLEVEL_STATION || picked.z == ZLEVEL_MINING || picked.z == ZLEVEL_SPACEMIN))

```

Comment:
z3 is the abandonned telecomsat not the derelict, but yeah, I can give it a specific name.